### PR TITLE
Implementing subscribe to sendgrid PHP example function

### PR DESCRIPTION
**Pull request by Shylajhaa**
Implementing subscribe to sendgrid PHP example function [#1915](https://github.com/appwrite/appwrite/issues/1915)

1. Created a function in project
Screenshot 2021-10-18 at 8 48 35 PM

2. Execute function with email as input
Screenshot 2021-10-18 at 8 47 57 PM

3. On successful execution message is printed to console
Screenshot 2021-10-18 at 8 48 18 PM

### CODE REVIEW

**Comment by christyjacob4** [comment]
Please attach screenshots of a successful execution along with the console output

File: php/subscribe-to-sendgrid/README.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+```bash
+$ cd ..
+$ tar -zcvf code.tar.gz subscribe-to-sendgrid
+```
+
+- Navigate to the Overview Tab of your Cloud Function > Deploy Tag
+- Input the command that will run your function (in this case "php index.php") as your entrypoint command
+- Upload your tarfile
+- Click 'Activate'
```
Comment (christyjacob4) [inline php/subscribe-to-sendgrid/README.md:41]: You can add the **Trigger** section back and mention that this function is triggered manually. Like this example here https://github.com/appwrite/demos-for-functions/pull/154/files#diff-8686b4afc9e25133e17a513bc8d56a4d49a0a2cfa8e23dd6436438d33fe1d618R47

**Review by christyjacob4 — CHANGES_REQUESTED** [review]
please address one last comment and we can merge this 😄 

Awesome work !

**Comment by Shylajhaa** [comment]
> please address one last comment and we can merge this 😄
> 
> Awesome work !

@christyjacob4 Done, we can merge now. Thanks :)

**Comment by christyjacob4** [comment]
Awesome work @Shylajhaa  👏

**Comment by christyjacob4** [comment]
THANK YOU! All changes merged 🥳

Please reach out to me on our [Discord server](https://appwrite.io/discord) if you would like to claim your Appwrite swags! As a way of saying thank you, we would also love to invite you to join the Appwrite organization on GitHub. Please share your GitHub username with us on Discord.  

You can accept the invite by visiting https://github.com/orgs/appwrite/invitation. By joining our team, you will officially be an Appwrite maintainer on GitHub.

You can change your membership [visibility settings](https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/publicizing-or-hiding-organization-membership), so your new Appwrite team membership badge will show up on your personal GitHub profile.

Please feel free to look for more PRs you might be interested in helping with on our long list of [Hacktoberfest friendly issues](https://github.com/search?q=org%3Aappwrite+org%3Autopia-php+is%3Aissue+label%3Ahacktoberfest+created%3A%3E2021-01-01+is%3Aopen&type=issues) and help make Appwrite better :)

**Pull request merged**